lib/registerpage/registerpage_form.dart

```diff
@@ -0,0 +1,250 @@
+                )
+              )
+            ),
+            Positioned(
+              top: widget.screenHeight * 0.43,
+              child: SizedBox(
+                height: widget.screenHeight * 0.08,
+                width: widget.screenWidth * 0.2,
+                child: TextField(
+                  decoration: InputDecoration(
+                    focusedBorder: OutlineInputBorder(
+                      borderSide: BorderSide(
+                        color: _outlineBorderColorPassword
+                      )
+                    ),
+                    enabledBorder: OutlineInputBorder(
+                      borderSide: BorderSide(
+                        color: _outlineBorderColorPassword
+                      )
+                    ),
+                    border: OutlineInputBorder(),
+                    hintText: "Password",
+                  ),
+                  controller: passwordController,
+                )
+              )
+            ),
+            Positioned(
+              top: widget.screenHeight * 0.52,
+              child: SizedBox(
+                height: widget.screenHeight * 0.08,
+                width: widget.screenWidth * 0.2,
+                child: TextField(
+                  decoration: InputDecoration(
+                      focusedBorder: OutlineInputBorder(
+                          borderSide: BorderSide(
+                          color: _outlineBorderColorPasswordSecond
+                        )
+                      ),
+                      enabledBorder: OutlineInputBorder(
+                        borderSide: BorderSide(
+                          color: _outlineBorderColorPasswordSecond
+                        )
+                      ),
+                    hintText: "Confirm Password"
+                  ),
+                  controller: secondPasswordController,
+                )
+              )
+            ),
+            Positioned(
+              top: widget.screenHeight * 0.61,
+              child: SizedBox(
+                height: widget.screenHeight * 0.06,
+                width: widget.screenWidth * 0.095,
+                child: RaisedButton(
+                  child: Text(
+                    "Sign up",
+                  style: TextStyle(fontSize: widget.screenWidth * 0.0115),
+                  ),
+                  color: Color.fromARGB(255, 42, 157, 143),
+                  onPressed: register,
+                ),
+              ),
+            ),
+            Positioned(
+              bottom: widget.screenHeight * 0.1,
+              child: Container(
+                width: widget.screenWidth * 0.15,
+                height: widget.screenHeight * 0.04,
+                child: TextButton(
+                  style: TextButton.styleFrom(
+                    textStyle: TextStyle(
+                      color: Color.fromARGB(255, 42, 157, 143),
+                    )
+                  ),
+                  onPressed: () {},
```

Comment:
Should you navigate here to the login page? I don't see anymore a point from where you access the login page.